WordPress-Extra/ruleset.xml

```diff
@@ -73,7 +73,9 @@
 	<!-- Verify modifier keywords for declared methods and properties in classes.
 		 https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/1101 -->
 	<rule ref="Squiz.Scope.MethodScope"/>
-	<rule ref="Squiz.Scope.MemberVarScope"/>
+	<rule ref="PSR2.Classes.PropertyDeclaration">
+		<exclude name="PSR2.Classes.PropertyDeclaration.Multiple"/>
```

Comment:
I've never seen any plugin or theme code declare multiple properties - I'm not even 100% sure that I knew it was possible!

Since this is in `Extra`, I would be fine with removing this exclusion, and letting those individual developers who do prefer this approach, to add their own exclusion.

Reply:
Let me check and see if WP Core has any multi-property assignments, if not, I'll change it.

Reply:
The results of a run against WP Core, while pretty embarrasing, does not yield any multi-property assignments, so I've removed the exclusion.

```

PHP CODE SNIFFER VIOLATION SOURCE SUMMARY
----------------------------------------------------------------------
SOURCE                                                           COUNT
----------------------------------------------------------------------
PSR2.Classes.PropertyDeclaration.ScopeMissing                    1466
PSR2.Classes.PropertyDeclaration.VarUsed                         1349
PSR2.Classes.PropertyDeclaration.Underscore                      366
----------------------------------------------------------------------
A TOTAL OF 3181 SNIFF VIOLATIONS WERE FOUND IN 3 SOURCES
----------------------------------------------------------------------
```